src/src/utils.h

```diff
@@ -17,17 +17,4 @@ long rng8Bit(void);
 long rng5Bit(void);
 
 // returns 0 <= x < n where n <= 256
-unsigned int rngN(unsigned int upper);
-
-
-#include <string.h>
-#include <stdint.h>
-#include <stdio.h>
-
-#define H1(s,i,x)   (x*65599u+(uint8_t)s[(i)<strlen(s)?strlen(s)-1-(i):strlen(s)])
-#define H4(s,i,x)   H1(s,i,H1(s,i+1,H1(s,i+2,H1(s,i+3,x))))
-#define H16(s,i,x)  H4(s,i,H4(s,i+4,H4(s,i+8,H4(s,i+12,x))))
-#define H64(s,i,x)  H16(s,i,H16(s,i+16,H16(s,i+32,H16(s,i+48,x))))
-#define H256(s,i,x) H64(s,i,H64(s,i+64,H64(s,i+128,H64(s,i+192,x))))
-
-#define HASH(s)    ((uint32_t)(H256(s,0,0)^(H256(s,0,0)>>16)))
\ No newline at end of file
+unsigned int rngN(unsigned int upper);
```

Comment:
No new line here. While it will compile fine, this will likely introduce a new warning in the build output, and we should be squashing our warnings, not adding more ;)